### ISSUE TITLE
Submit Phase 1 data

### ISSUE THREAD

**Issue by NewGraphEnvironment** [NewGraphEnvironment/fish_passage_fraser_2023_reporting#108]
Unfortunately we need a windows machine to QA the final spreadsheet, package and submit (from all I can tell anyway).

I have machine so Lucy if you are able to get the spreadsheet ready and photos to ~/Library/CloudStorage/OneDrive-Personal/Projects/submissions/PSCIS/2023/fraser I can take it from there

I will link to here with csv export from the two projects and script used to generate it.

**Comment by NewGraphEnvironment** [comment]
just changed name of export submission (as above) to match purpose of last linked script.  Here is the latest version of the script to get all the photos into one place https://github.com/NewGraphEnvironment/fish_passage_skeena_2023_reporting/blob/main/scripts/01_prep_inputs/0140-pscis-export-submission.R

of course will need to be run twice to get photos from both QGIS projects vs just the one as in the Skeena

**Comment by NewGraphEnvironment** [comment]
Actually - here is the last time an export was done.  it was simplified with `fs` calls and comes from just the `reassessments` section of [fish_passage_moti_2022_reporting](https://github.com/NewGraphEnvironment/fish_passage_moti_2022_reporting/blob/main/scripts/01_prep_data/0118-pscis-submission-extract.R)



```
## Reassessments-----------------------------------------------------------------
# this was updated July 2024 using Skeena 2023 as template
library(fpr)
library(tidyverse)
library(fs)


# PSCIS Submissions -------------

tfpr_filter_list <- function(idx){
  filestocopy_list[idx]
}

tfpr_photo_change_name <- function(filenames_to_change = filestocopy_list){
  gsub(filenames_to_change, pattern = path, replacement = targetdir)
}


name_repo <- 'fish_passage_moti_2022_reporting'
name_pdf <- 'Moti2022.pdf'
url_github <- 'https://github.com/NewGraphEnvironment/'
url_gitpages <- 'https://newgraphenvironment.github.io/'
name_submission <- 'pscis_reassessments.xlsm'


# need to add photos to local machine to upload to PSCIS
targetdir = fs::path('/Users/airvine/Library/CloudStorage/OneDrive-Personal/Projects/PSCIS/2022/reassessments', name_repo)

d <- fpr::fpr_import_pscis(workbook_name = 'pscis_reassessments.xlsm')

folderstocopy<- d$pscis_crossing_id %>% as.character()

path <- fs::path_wd('data/photos/')

path_to_photos <- fs::path(path, folderstocopy)

# here we transfer just the photos with labels over into the PSCIS directory where we will upload from to the gov interface

fs::dir_create(targetdir)

folderstocreate<- fs::path(targetdir, folderstocopy)

##create the folders
fs::dir_create(folderstocreate)


filestocopy_list <- path_to_photos %>%
  purrr::map(fpr::fpr_photo_paths_to_copy) %>%
  purrr::set_names(basename(folderstocreate))


##view which files do not have any photos to paste by reviewing the empty_files object
empty_idx <- which(!lengths(filestocopy_list))
empty_files <- empty_idx %>% tfpr_filter_list()

##rename long names if necessary

photo_sort_tracking <- path_to_photos %>%
  purrr::map(fpr::fpr_photo_document_all) %>%
  purrr::set_names(folderstocopy) %>%
  bind_rows(.id = 'folder') %>%
  mutate(photo_name = str_squish(str_extract(value, "[^/]*$")),
         photo_name_length = stringr::str_length(photo_name))

###here we back up a csv that gives us the new location and name of the original JPG photos.
## Not ideal because we did some sorting by hand without adding name of camera to the file name but a start on reproducibility nonetheless

##burn to csv
photo_sort_tracking %>%
  readr::write_csv(file = 'data/photos/photo_sort_tracking_reassessments.csv')

filestopaste_list <- filestocopy_list %>%
  map(tfpr_photo_change_name)

##!!!!!!!!!!!!!!!copy over the photos!!!!!!!!!!!!!!!!!!!!!!!
mapply(fs::file_copy,
       path =  filestocopy_list,
       new_path = filestopaste_list)


##also move over the pscis file
fs::file_copy(path = fs::path('data', name_submission),
              new_path = fs::path(targetdir, name_submission),
              overwrite = T)

#make a little readme for the pdf for upload to ecocat and other details
writeLines(
  paste(
    "Online interactive report is located at: ",
    paste0(url_gitpages, name_repo),
    "",
    "A versioned pdf of the report can be downloaded from: ",
    paste0(url_github, name_repo, "/raw/main/docs/", name_pdf),
    "",
    "Raw data is available here: ",
    paste0(url_github, name_repo, "/tree/main/data"),
    "",
    "All scripts to produce online interactive reporting and pdf are located at: ",
    paste0(url_github, name_repo),
    sep = "\n"
  ),
  fs::path(targetdir, 'readme.txt')
)

##in the future we will need to add the copy calls to move the directory off of onedrive to
#  the windows machine used for project submission.  Don't want to set up the machine right now
# so will do it by hand. What an insane pain
```

**Comment by lucy-schick** [comment]
cool. I'll get on it

**Comment by lucy-schick** [comment]
csvs have time attached to the date, we will have to fix and reburn to csv.

![Screen Shot 2024-09-13 at 3 44 05 PM](https://github.com/user-attachments/assets/b7ea245f-948c-41f5-9fc8-e42881d61f0e)

**Comment by lucy-schick** [comment]
nevermind! found the date column! dont use `date_time_start`, use `date`

![Screen Shot 2024-09-13 at 3 46 43 PM](https://github.com/user-attachments/assets/4f8abba4-f1f5-41ae-8e98-0aee0beed7ff)

**Comment by NewGraphEnvironment** [comment]
submission uploaded to ESF as submitssion id: 2369231

wow.

classic PSCIS submission absolute time suck insanity.

here is a screenshot of the xml file errors overcome. the errors related to the crossing subtype make sense (although the excel validation missed them) as I had accidentally erased "Ford" from the submission spreadsheet before copy pasting from .csv to the .xlsx template.  fixed these before rezip and new xml file generation

The first error on ESF upload though makes no sense. For some reason `50` from the `Percent Backwatered' field of crossing `13903618` did not transfer from the `xlsx` file to the `.xml` during the `Generate Submission` phase of the upload.  Not sure at all why. To overcome needed to:

1. use error message info to track down where the problem was (ie. row 6356 of the xml file).
2. use proper text editor (sublime text - maybe rstudio could have worked too) to edit the xml file to include `50` in the xml file
3. resubmit with the hand bombed xml file


Even though it is "accepted" it may not land.  We have had many many many long and painfull battles with this system over the years trying to get data to land so we may very well recieve cryptic error messages tomorrow morning that it didn't land. fingers crossed though. will close issue if it does land



<img width="1510" alt="image" src="https://github.com/user-attachments/assets/14282f5c-9f0e-44e5-9ebd-da56be87590c">

**Comment by lucy-schick** [comment]
ok so here are the errors I get. I couldn't find the screenshots I took on the PC so sorry for the poor quality. 

Last 4 I fixed  with adding in ford as per above. 

Looks like the backwater issue is still there as well. I will fix that.

I'll look into the other issues tomorrow and get it submitted tomorrow. Did my best today but just ran out of time. 

![errors](https://github.com/user-attachments/assets/f36a2c3d-a336-4d9a-8403-cc85443c69a3)

**Comment by lucy-schick** [comment]
PSCIS data has been submitted. 

Above issues were fixed by:
1. Adding in Funding source as OTHER
2. Adding in the budget amount, i used 30,000 which is what it said in the 2023 skeena phase 1 spreadsheet.
3. As per above, Added in Ford.
4. As per above, Edited the XML file in Sublime text and changed backwatered to 50. See first screenshot. 

![Screen Shot 2024-10-28 at 10 23 21 AM](https://github.com/user-attachments/assets/d8b1c303-1852-4094-a30c-abe1fbb7358c)


![{32D21A9F-058B-402C-90C0-311D41A7C963}](https://github.com/user-attachments/assets/670de2ea-2eab-4d51-b27f-a13d51d3725e)

**Comment by NewGraphEnvironment** [comment]
awesome. Thanks! 

If we need to do it again (maybe likely - but hopefully not) we can put MoTi for the funder.   Think the submissions were made from the `Documents/submissions` (or something like that) directory on the PC so _I think_ the old submission should be there for reference

It is incredibly odd that the backwater error reappears after a fresh copy and paste to the template. Makes me think it must be the template causing the issue...  Can point them to this issue eventually once we hopefully get the data to land....

**Comment by lucy-schick** [comment]
20241028 Submission got rejected. Below are the errors:

2 errors unrelated to photos are documented here https://github.com/NewGraphEnvironment/fish_passage_fraser_2023_reporting/issues/113#issuecomment-2445358469 and https://github.com/NewGraphEnvironment/fish_passage_fraser_2023_reporting/issues/113#issuecomment-2445366318. ChatGPT was helpful for finding these amid all the text, https://chatgpt.com/share/e/67217347-a180-8012-afb8-cb6cd3c069f0


Crossing: 13903627 Attachment: TimePhoto_20231004_093006_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903451 Attachment: TimePhoto_20231003_132949_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200051 Attachment: TimePhoto_20231005_155342_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900308 Attachment: TimePhoto_20231003_125007_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903148 Attachment: TimePhoto_20231005_094718_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 5400019. The supplied value of 99.9 is not valid for Structure Length. The value supplied must be >0 and <=99.0.

Crossing: 13900003 Attachment: TimePhoto_20231005_110423_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900015 Attachment: TimePhoto_20231005_095647_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900025 Attachment: TimePhoto_20231005_085616_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900073 Attachment: TimePhoto_20231005_122353_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903148 Attachment: TimePhoto_20231005_094047_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900201 Attachment: TimePhoto_20231004_142633_road.JPG. The process was unable to resize th More Info

2024-10-28 10:34:32
e attachment photos down to the required size of 1024X1024.
Crossing: 9900446 Attachment: TimePhoto_20230930_152258_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900003 Attachment: TimePhoto_20231005_111108_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900193 Attachment: TimePhoto_20231004_112913_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900265 Attachment: TimePhoto_20231004_115112_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900309 Attachment: TimePhoto_20231003_120850_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900015 Attachment: TimePhoto_20231005_095712_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900270 Attachment: TimePhoto_20231004_123347_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900073 Attachment: TimePhoto_20231005_122822_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900050 Attachment: TimePhoto_20231003_163405_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900064 Attachment: TimePhoto_20231004_172525_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900030 Attachment: TimePhoto_20231005_114229_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.
More Info

2024-10-28 10:34:33
>
Crossing: 13900252 Attachment: TimePhoto_20231003_144816_o_utlet2.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900064 Attachment: TimePhoto_20231004_171418_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900019 Attachment: TimePhoto_20231004_155945_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900012 Attachment: TimePhoto_20231005_125512_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900012 Attachment: TimePhoto_20231005_124740_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900201 Attachment: TimePhoto_20231004_143718_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900003 Attachment: TimePhoto_20231005_111332_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900043 Attachment: TimePhoto_20231003_173308_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900201 Attachment: TimePhoto_20231004_142957_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22201229 Attachment: TimePhoto_20231005_142818_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900077 Attachment: TimePhoto_20231005_104401_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900309 Attachment: TimePhoto_20231003_1 More Info

2024-10-28 10:34:33
21000_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.
Crossing: 9900446 Attachment: TimePhoto_20230930_153226_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900193 Attachment: TimePhoto_20231004_111948_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900026 Attachment: TimePhoto_20231003_113805_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900077 Attachment: TimePhoto_20231005_104345_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200022 Attachment: TimePhoto_20231005_134123_o_utletembankment.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903184 Attachment: TimePhoto_20231004_100239_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903450 Attachment: TimePhoto_20231003_134731_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900026 Attachment: TimePhoto_20231003_113631_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903627 Attachment: TimePhoto_20231004_093501_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900309 Attachment: TimePhoto_20231003_121954_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200022 Attachment: TimePhoto_20231005_133512_upstream.JPG. The process was unable to res More Info

2024-10-28 10:34:33
ize the attachment photos down to the required size of 1024X1024.
Crossing: 13900198 Attachment: TimePhoto_20231004_132808_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900308 Attachment: TimePhoto_20231003_124706_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200029 Attachment: TimePhoto_20231005_151216_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903148 Attachment: TimePhoto_20231005_093630_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900270 Attachment: TimePhoto_20231004_123002_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200029 Attachment: TimePhoto_20231005_151655_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900019 Attachment: TimePhoto_20231004_161233_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200051 Attachment: TimePhoto_20231005_155234_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900193 Attachment: TimePhoto_20231004_112128_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900265 Attachment: TimePhoto_20231004_115325_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900052 Attachment: TimePhoto_20231003_152738_downstream.JPG. The process was unable to resize the attachment photos down to the require More Info

2024-10-28 10:34:33
d size of 1024X1024.
Crossing: 13905581 Attachment: TimePhoto_20231003_160503_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900052 Attachment: TimePhoto_20231003_153228_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903184 Attachment: TimePhoto_20231004_095801_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 24715750 Attachment: TimePhoto_20231003_110932_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900064 Attachment: TimePhoto_20231004_172533_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900050 Attachment: TimePhoto_20231003_163258_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900019 Attachment: TimePhoto_20231004_160538_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900015 Attachment: TimePhoto_20231005_100220_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900192 Attachment: TimePhoto_20231004_103945_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 24715750 Attachment: TimePhoto_20231003_111011_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903451 Attachment: TimePhoto_20231003_132650_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903184 A More Info

2024-10-28 10:34:33
ttachment: TimePhoto_20231004_100514_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.
Crossing: 13900052 Attachment: TimePhoto_20231003_153312_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900265 Attachment: TimePhoto_20231004_120542_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200051 Attachment: TimePhoto_20231005_155200_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900077 Attachment: TimePhoto_20231005_105119_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900192 Attachment: TimePhoto_20231004_104050_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900043 Attachment: TimePhoto_20231003_171551_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900201 Attachment: TimePhoto_20231004_144839_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900043 Attachment: TimePhoto_20231003_171517_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900064 Attachment: TimePhoto_20231004_170347_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200051 Attachment: TimePhoto_20231005_155252_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900050 Attachment: TimePhoto_20231003_163624_barrel.JPG. The proces More Info

2024-10-28 10:34:33
s was unable to resize the attachment photos down to the required size of 1024X1024.
Crossing: 13900052 Attachment: TimePhoto_20231003_152729_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900265 Attachment: TimePhoto_20231004_115604_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903627 Attachment: TimePhoto_20231004_093550_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903451 Attachment: TimePhoto_20231003_132642_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900265 Attachment: TimePhoto_20231004_120110_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900308 Attachment: TimePhoto_20231003_130626_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900073 Attachment: TimePhoto_20231005_122509_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900193 Attachment: TimePhoto_20231004_111507_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200022 Attachment: TimePhoto_20231005_134209_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900201 Attachment: TimePhoto_20231004_143157_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200029 Attachment: TimePhoto_20231005_151115_road.JPG. The process was unable to resize the attachment photos down to the required More Info

2024-10-28 10:34:33
size of 1024X1024.
Crossing: 13900025 Attachment: TimePhoto_20231005_090133_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900019 Attachment: TimePhoto_20231004_161840_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900025 Attachment: TimePhoto_20231005_085718_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900193 Attachment: TimePhoto_20231004_112612_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900193 Attachment: TimePhoto_20231004_112945_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903450 Attachment: TimePhoto_20231003_135306_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900077 Attachment: TimePhoto_20231005_104112_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900252 Attachment: TimePhoto_20231003_144808_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900270 Attachment: TimePhoto_20231004_124212_i_nletpic2.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900003 Attachment: TimePhoto_20231005_110348_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900192 Attachment: TimePhoto_20231004_104031_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 9900446 Attachm More Info

2024-10-28 10:34:33
ent: TimePhoto_20230930_153734_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.
Crossing: 13900073 Attachment: TimePhoto_20231005_122523_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900030 Attachment: TimePhoto_20231005_114329_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22201229 Attachment: TimePhoto_20231005_143446_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903450 Attachment: TimePhoto_20231003_135805_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900043 Attachment: TimePhoto_20231003_172225_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900192 Attachment: TimePhoto_20231004_105825_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900019 Attachment: TimePhoto_20231004_160307_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13905581 Attachment: TimePhoto_20231003_160502_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900198 Attachment: TimePhoto_20231004_132901_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900200 Attachment: TimePhoto_20231004_135856_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900198 Attachment: TimePhoto_20231004_130535_road.JPG. The process was More Info

2024-10-28 10:34:33
unable to resize the attachment photos down to the required size of 1024X1024.
Crossing: 13900270 Attachment: TimePhoto_20231004_124249_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900073 Attachment: TimePhoto_20231005_122754_weirdownstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900012 Attachment: TimePhoto_20231005_125535_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900026 Attachment: TimePhoto_20231003_113230_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903184 Attachment: TimePhoto_20231004_095634_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900019 Attachment: TimePhoto_20231004_162503_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903148 Attachment: TimePhoto_20231005_093742_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900003 Attachment: TimePhoto_20231005_110210_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 9900446 Attachment: TimePhoto_20230930_152407_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900252 Attachment: TimePhoto_20231003_145421_i_nlet2.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900270 Attachment: TimePhoto_20231004_123318_downstream.JPG. The process was unable to resize the attachment photos down to More Info

2024-10-28 10:34:33
the required size of 1024X1024.
Crossing: 22200022 Attachment: TimePhoto_20231005_133234_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22201229 Attachment: TimePhoto_20231005_143314_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903627 Attachment: TimePhoto_20231004_093019_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900012 Attachment: TimePhoto_20231005_131631_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900064 Attachment: TimePhoto_20231004_170928_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13905581 Attachment: TimePhoto_20231003_160553_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200051 Attachment: TimePhoto_20231005_155357_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900030 Attachment: TimePhoto_20231005_114351_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200022 Attachment: TimePhoto_20231005_134131_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900050 Attachment: TimePhoto_20231003_163454_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900026 Attachment: TimePhoto_20231003_113154_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900309 At More Info

2024-10-28 10:34:34
tachment: TimePhoto_20231003_121228_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.
Crossing: 22201229 Attachment: TimePhoto_20231005_142655_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900309 Attachment: TimePhoto_20231003_121923_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903450 Attachment: TimePhoto_20231003_134907_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900043 Attachment: TimePhoto_20231003_171719_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900200 Attachment: TimePhoto_20231004_140405_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900064 Attachment: TimePhoto_20231004_170950_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900030 Attachment: TimePhoto_20231005_114315_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900003 Attachment: TimePhoto_20231005_111158_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200029 Attachment: TimePhoto_20231005_151636_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22201229 Attachment: TimePhoto_20231005_143709_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200029 Attachment: TimePhoto_20231005_151200_outlet.JPG. More Info

2024-10-28 10:34:34
The process was unable to resize the attachment photos down to the required size of 1024X1024.
Crossing: 13900026 Attachment: TimePhoto_20231003_113014_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900198 Attachment: TimePhoto_20231004_131518_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900200 Attachment: TimePhoto_20231004_140333_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900030 Attachment: TimePhoto_20231005_114433_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900073 Attachment: TimePhoto_20231005_122204_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900252 Attachment: TimePhoto_20231003_144758_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900050 Attachment: TimePhoto_20231003_163944_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900252 Attachment: TimePhoto_20231003_144306_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900309 Attachment: TimePhoto_20231003_122656_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900052 Attachment: TimePhoto_20231003_152526_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900270 Attachment: TimePhoto_20231004_124159_upstream.JPG. The process was unable to resize the attachment photos down More Info

2024-10-28 10:34:34
to the required size of 1024X1024.
Crossing: 13900201 Attachment: TimePhoto_20231004_145203_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900308 Attachment: TimePhoto_20231003_125528_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13905581 Attachment: TimePhoto_20231003_155842_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900192 Attachment: TimePhoto_20231004_105222_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903184 Attachment: TimePhoto_20231004_101327_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900025 Attachment: TimePhoto_20231005_085743_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900077 Attachment: TimePhoto_20231005_104332_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200022 Attachment: TimePhoto_20231005_133948_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903451 Attachment: TimePhoto_20231003_132353_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903627 Attachment: TimePhoto_20231004_093239_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900050 Attachment: TimePhoto_20231003_163613_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Cros More Info

2024-10-28 10:34:34
sing: 13900077 Attachment: TimePhoto_20231005_105114_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.
Crossing: 9900446 Attachment: TimePhoto_20230930_151848_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903450 Attachment: TimePhoto_20231003_135825_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903450 Attachment: TimePhoto_20231003_134851_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903148 Attachment: TimePhoto_20231005_093052_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903451 Attachment: TimePhoto_20231003_132152_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903148 Attachment: TimePhoto_20231005_093342_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 22200022 Attachment: TimePhoto_20231005_133505_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 9900446 Attachment: TimePhoto_20230930_153602_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900200 Attachment: TimePhoto_20231004_135425_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900012 Attachment: TimePhoto_20231005_125244_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900015 Attachment: TimePhoto_20231005_095732_upstream.JPG. Th More Info

2024-10-28 10:34:34
e process was unable to resize the attachment photos down to the required size of 1024X1024.
Crossing: 22200051 Attachment: TimePhoto_20231005_155239_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900198 Attachment: TimePhoto_20231004_131642_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 24715750 Attachment: TimePhoto_20231003_111005_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903627 Attachment: TimePhoto_20231004_092713_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900025 Attachment: TimePhoto_20231005_090938_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900012 Attachment: TimePhoto_20231005_124756_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 24715750 Attachment: TimePhoto_20231003_111031_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900030 Attachment: TimePhoto_20231005_114452_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900252 Attachment: TimePhoto_20231003_144421_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900052 Attachment: TimePhoto_20231003_152750_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900015 Attachment: TimePhoto_20231005_095541_road.JPG. The process was unable to resize the attachment photos dow More Info

2024-10-28 10:34:34
n to the required size of 1024X1024.
Crossing: 13900026 Attachment: TimePhoto_20231003_113706_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900025 Attachment: TimePhoto_20231005_090825_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900073 Attachment: TimePhoto_20231005_121947_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13905581 Attachment: TimePhoto_20231003_160616_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900308 Attachment: TimePhoto_20231003_124309_road.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903451 Attachment: TimePhoto_20231003_132258_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900192 Attachment: TimePhoto_20231004_104655_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900252 Attachment: TimePhoto_20231003_144408_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900015 Attachment: TimePhoto_20231005_100238_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900198 Attachment: TimePhoto_20231004_132736_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13903184 Attachment: TimePhoto_20231004_101131_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 139 More Info

2024-10-28 10:34:34
00043 Attachment: TimePhoto_20231003_173048_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.
Crossing: 22201229 Attachment: TimePhoto_20231005_142633_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900270 Attachment: TimePhoto_20231004_123557_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900252 Attachment: TimePhoto_20231003_144442_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900265 Attachment: TimePhoto_20231004_115338_downstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900308 Attachment: TimePhoto_20231003_124424_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 24715750 Attachment: TimePhoto_20231003_110951_inlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 13900200 Attachment: TimePhoto_20231004_140317_outlet.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 24715750 Attachment: TimePhoto_20231003_110955_upstream.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

Crossing: 5400024. The supplied value of 12.0 is not valid for Structure Diameter or Span. If stream crossing type is Closed Bottom Structure then the value supplied must be > 0 and <=9.99 meters. If stream crossing type is Open Bottom Structure then the value supplied must be > 0 and <=300 meters. If the crossing type is Other than the value is not required.

Crossing: 22200029 Attachment: TimePhoto_20231005_ More Info

2024-10-28 10:34:34
153139_barrel.JPG. The process was unable to resize the attachment photos down to the required size of 1024X1024.

**Comment by NewGraphEnvironment** [comment]
interesting that there were 53 less errors... That was the result only the 1 additional photo that was renamed (by me) in the most recent submission vs the one before? Perhaps reply in issue if that assumption is incorrect


made `fpr::fpr_photo_qa2` to test for dupes. non found.

logged metadata strip to reduce metadata columns of all photos (sometime over 100 to 20 or so) with following common warning logged (still works just issues warning):

`"Warning: ICC_Profile deleted. Image colors may be affected"`

Interestingly this occured exactly 223 times out of the 1113 photos processed. Doesn't quite match our number of errors on the last submission attempt but sort of close.  

reprocessed photos are in 

`~/Library/CloudStorage/OneDrive-Personal/Projects/submissions/PSCIS/2023/fraser/photo_meta_rm`

will point here with latest scripts to reproduce as well as log of strip details

**Comment by lucy-schick** [comment]
I changed probably 5 more photos after you did, so there was definitely more than 1 photo renamed between yesterday’s submission and Friday.

**Comment by lucy-schick** [comment]
20241031 Submission is submitted. Fixes include:
- hacked XML to fix backwatering issue
- photos stripped of meta data

![{3CCC1A19-B939-43E1-BEBB-B2937422D398}](https://github.com/user-attachments/assets/4afe7b90-8b2c-4282-bd13-3e5fb3fc977b)

**Comment by lucy-schick** [comment]
Data has been successfully submitted.

Closing in favour of
- metadata strip https://github.com/NewGraphEnvironment/fish_passage_guidebook/issues/13
- reserve photo duplicates https://github.com/NewGraphEnvironment/fish_passage_guidebook/issues/12
